src/Components/Members/MembersCreateEdit.js

```diff
@@ -0,0 +1,316 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import '../FormStyles.css';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import TextField from '@mui/material/TextField';
+import Card from '@mui/material/Card';
+import Button from '@mui/material/Button';
+import CardHeader from '@mui/material/CardHeader';
+import { Alert } from '@mui/material';
+import { useDropzone } from 'react-dropzone';
+
+const thumb = {
+  display: 'inline-flex',
+  borderRadius: 2,
+  border: '1px solid #eaeaea',
+  margin: 'auto',
+  width: 'auto',
+  height: 60,
+  padding: 4,
+  boxSizing: 'border-box',
+};
+
+function MembersCreateEdit() {
+  const { id } = useParams();
+  const urlEdit = `http://ongapi.alkemy.org/public/api/members/${id}`;
+  const urlCreate = `http://ongapi.alkemy.org/public/api/members`;
+  const [initialValues, setInitialValues] = useState({
+    name: '',
+    image: '',
+    description: '',
+    facebookUrl: '',
+    linkedinUrl: '',
+  });
+
+  const [files, setFiles] = useState([]);
+  const { getRootProps, getInputProps } = useDropzone({
+    accept: 'image/png, image/jpg',
+    onDrop: (acceptedFiles) => {
+      setInitialValues({
+        ...initialValues,
+        image: acceptedFiles.map((file) =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file),
+          }),
+        ),
+      });
+      setFiles(
+        acceptedFiles.map((file) =>
+          Object.assign(file, {
+            preview: URL.createObjectURL(file),
+          }),
+        ),
+      );
+    },
+  });
+  const thumbs = files.map((file) => (
+    <div key={file.name} style={thumb}>
+      <div className="thumb-inner">
+        <img className="img" src={file.preview} />
+      </div>
+    </div>
+  ));
+
+  useEffect(
+    () => () => {
+      // Make sure to revoke the data uris to avoid memory leaks
+      files.forEach((file) => URL.revokeObjectURL(file.preview));
+    },
+    [files],
+  );
+
+  const descriptionChange = (e, editor) => {
+    const data = editor.getData();
+
+    setInitialValues({ ...initialValues, description: data });
+  };
+  const nameChange = (e, values) => {
+    // eslint-disable-next-line no-console
+    console.log(values);
+    setInitialValues({ ...initialValues, name: e.target.value });
+  };
+  const facebookChange = (e) => {
+    setInitialValues({ ...initialValues, facebookUrl: e.target.value });
+  };
+  const linkedInChange = (e) => {
+    setInitialValues({ ...initialValues, linkedinUrl: e.target.value });
+  };
```

Comment:
for name, facebook and linkedin you should create only one onChange and use a switch on it